lib/task/migrate/migrations/arMigration0166.class.php

```diff
@@ -18,7 +18,7 @@
  */
 
 /*
- * Add column to determine whether a menu item is deleteable
+ * Store info related to menu locking
```

Comment:
Changing this migration may break some databases, not a huge deal in this qa state, but I'd add a note in the PR and the Redmine ticket.